packages/cli/test/controller.js

```diff
@@ -5,11 +5,16 @@
 
 'use strict';
 
+const expect = require('@loopback/testlab').expect;
+const debug = require('../lib/debug')('controller-test');
 const path = require('path');
 const assert = require('yeoman-assert');
 const helpers = require('yeoman-test');
 const fs = require('fs');
+const util = require('util');
+const utils = require('./test-utils');
```

Comment:
The difference between `util` vs. `utils` is easy to overlook. Could you please rename the second import from `utils` to `testUtils`?